src/makeInfinityAdder.js

```diff
@@ -4,7 +4,20 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(number) {
+    if (number === undefined) {
```

Comment:
The condition `if (number === undefined)` is used to reset the sum. This works, but it might be more explicit to check if `arguments.length === 0` to ensure that the function is called without any arguments, rather than relying on `undefined`.